src/rules/labelPositionRule.ts

```diff
@@ -29,10 +29,10 @@ class LabelPositionWalker extends Lint.RuleWalker {
     public visitLabeledStatement(node: ts.LabeledStatement) {
         const statement = node.statement;
         if (statement.kind !== ts.SyntaxKind.DoStatement
-                && statement.kind !== ts.SyntaxKind.ForStatement
-                && statement.kind !== ts.SyntaxKind.ForInStatement
-                && statement.kind !== ts.SyntaxKind.WhileStatement
-                && statement.kind !== ts.SyntaxKind.SwitchStatement) {
+            && statement.kind !== ts.SyntaxKind.ForStatement
+            && statement.kind !== ts.SyntaxKind.ForInStatement
+            && statement.kind !== ts.SyntaxKind.WhileStatement
+            && statement.kind !== ts.SyntaxKind.SwitchStatement) {
```

Comment:
The downside of this alignment is that the conditions kind of blend in with the statements inside the if block. Assuming this is part of the indentation feature as well?